docs/topics/proc_updating-booster-deployment-configuration.adoc

```diff
@@ -0,0 +1,81 @@
+[id='proc_updating-booster-deployment-configuration_{context}']
+= Updating the Deployment Configuration of a Booster
+
+The deployment configuration for a booster contains information related to deploying and running the booster in OpenShift, such as route information or readiness probe location. 
+The deployment configuration of a booster is stored in a set of YAML files. 
+For boosters that use the Fabric8 Maven Plugin, the YAML files are located in the `src/main/fabric8/` directory. For boosters using Nodeshift, the YAML files are located in the `.nodeshift` directory.
+
+[IMPORTANT]
+====
+The deployment configuration files used by the Fabric8 Maven Plugin and Nodeshift do not have to be full OpenShift resource definitions. 
+Both Fabric8 Maven Plugin and Nodeshift can take the provided files and infer some missing information to create a full OpenShift resource definition.
+The resource definitions generated by the Fabric8 Maven Plugin are available in the `target/classes/META-INF/fabric8/` directory.
+The resource definitions generated by Nodeshift are available in the `tmp/nodeshift/resource/` directory.
+====
+
+
+.Prerequisite
```

Comment:
Sorry, missed that before. Should be _Prerequisites_ as there are more than one.